src/Object/Post.php

```diff
@@ -498,7 +498,7 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 
 		$tmp_item = [
 			'parentguid'      => $parent_guid,
-			'isreplyto'       => DI::l10n()->t('is reply to %s', $parent_username),
+			'isreplyto'       => DI::l10n()->t('in reply to %s', $parent_username),
```

Comment:
I just realized that the function is named `isreplyto` as well.

Reply:
Heh.